src/lib/hooks/routing/useClientSideSmartOrderRouterTrade.ts

```diff
@@ -10,7 +11,16 @@ import { getClientSideQuote } from './clientSideSmartOrderRouter'
 import { useRoutingAPIArguments } from './useRoutingAPIArguments'
 
 const protocols: Protocol[] = [Protocol.V2, Protocol.V3]
-const config = { protocols }
+
+// The following configs are used to reduce latency when using the client side SOR
+const MIN_DISTRIBUTION_PERCENT = 5
```

Comment:
This seems to be a regression from the defaults for some chains. If this defaults to 5, maybe we should just leave it?

See https://github.com/Uniswap/smart-order-router/blob/b19ebcb3f3e2b6b10a8021884f5336c8735ba8a5/src/routers/alpha-router/config.ts#L4